tests/Microsoft.Fx.Portability.Tests/ReadWriteApiPortOptionsTests.cs

```diff
@@ -67,6 +67,8 @@ private class TestAssemblyFile : IAssemblyFile
                 public string Version { get; } = string.Empty;
 
                 public Stream OpenRead() => File.OpenRead(Name);
+
+                public bool SkipBinaryIfPackageExists { get { return false; } }
```

Comment:
Please stay consistent with the rest of the file; `SkipBinaryIfPackagesExist => false;`

Reply:
Done